src/main/java/com/google/cloud/resourcemanager/OrganizationInfo.java

```diff
@@ -0,0 +1,312 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.resourcemanager;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.api.core.ApiFunction;
+import com.google.api.services.cloudresourcemanager.model.Organization;
+import com.google.cloud.StringEnumType;
+import com.google.cloud.StringEnumValue;
```

Comment:
The docs claim this "This represents a specific instance of a concept having a known set of acceptable String values, which can expand later due to new API features. Standard Java enums can't be used in such a context." though I'm not sure exactly why that's true. New constants can be added to  regular enum. What am I missing?

Reply:
we can use regular enum here if we want,but i prefare StringEnumValue just because of https://cloud.google.com/resource-manager/reference/rest/v1/organizations#LifecycleState

Reply:
Can you elaborate? That's about the JSON we're sending. I don't see how that impacts the decision to use or not use an enum here. I'm probably missing something obvious.